src/PlaywrightSharp.Chromium/ChromiumBrowser.cs

```diff
@@ -39,35 +37,46 @@ internal ChromiumBrowser(IBrowserApp app, ChromiumConnection connection, string[
             _connection.Disconnected += (sender, e) => Disconnected?.Invoke(this, EventArgs.Empty);
         }
 
-        /// <inheritdoc cref="IBrowser"/>
+        /// <inheritdoc cref="IBrowser.TargetChanged"/>
         public event EventHandler<TargetChangedArgs> TargetChanged;
 
-        /// <inheritdoc cref="IBrowser"/>
+        /// <inheritdoc cref="IBrowser.TargetCreated"/>
         public event EventHandler<TargetChangedArgs> TargetCreated;
 
-        /// <inheritdoc cref="IBrowser"/>
+        /// <inheritdoc cref="IBrowser.TargetDestroyed"/>
         public event EventHandler<TargetChangedArgs> TargetDestroyed;
 
-        /// <inheritdoc cref="IBrowser"/>
+        /// <inheritdoc cref="IBrowser.Disconnected"/>
         public event EventHandler Disconnected;
 
-        /// <inheritdoc cref="IBrowser"/>
-        public IBrowserContext[] BrowserContexts => null;
+        /// <inheritdoc cref="IBrowser.BrowserContexts"/>
+        public IEnumerable<IBrowserContext> BrowserContexts
+        {
+            get
+            {
+                yield return DefaultContext;
+

```

Comment:
remove extra line

Reply:
I don't consider this an extra line. I like the separation between the yield and the foreach.